docs/getting-started.rst

```diff
@@ -0,0 +1,76 @@
+===============
+Getting Started
+===============
+
+This page shows you how to get started with the :ref:`CrateDB PHP PDO client
+library <index>`.
+
+Prerequisites
+=============
+
+You need to be using PHP and `Composer`_.
+
+Install
+=======
+
+Install the library by adding it manually to your ``composer.json``:
+
+.. code-block:: json
+
+   {
+     "require": {
+       "crate/crate-pdo":"~0.6.0"
```

Comment:
Current version is 0.7.0 (https://github.com/crate/crate-pdo/commit/074f7af649f62f8070818491636f7e0087b90be8).